pyproject.toml

```diff
@@ -25,6 +26,7 @@ sentry-sdk = {version = "^1.2.0", extras = ["flask"]}
 athena = {git = "https://github.com/filipzz/athena.git", rev = "v0.8.3"}
 psycopg2-binary = "~2.8.6"
 consistent-sampler = {git = "https://github.com/votingworks/consistent_sampler.git"}
+scipy = "~1.6.0"
```

Comment:
this was missing even though we rely on it

Reply:
huh.